### PR TITLE
Recipe page

### PR DESCRIPTION
**Pull request by LateefahCamacho**

### CODE REVIEW

File: recipe/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+
+    <div>
+      <h2>Other Margarita Recipes</h2>
+      <a href="https://www.gimmesomeoven.com/cranberry-margaritas/"> Cranberry Margarita</a><br>
+      <a href="https://www.liquor.com/recipes/margarita/"> Classic Margarita</a>
+    </div>
+  </div>
+
+  <script src="script.js"></script>
```
Comment (nmestrada) [inline recipe/index.html]: I would remove any unnecessary code for readability

File: recipe/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  <!--
+  This script places a badge on your repl's full-browser view back to your repl's cover
+  page. Try various colors for the theme: dark, light, red, orange, yellow, lime, green,
+  teal, blue, blurple, magenta, pink!
+  -->
+  <script src="https://replit.com/public/js/replit-badge.js" theme="blue" defer></script>
```
Comment (nmestrada) [inline recipe/index.html]: Is this script used at all? If you remove it does it change the look of your page? If it is not used you can remove the comment and script tag

**Review by nmestrada — CHANGES_REQUESTED** [review]
looks good just remove the unused script tags 😄

File: recipe/index.html (hunk 2 above)
Comment (LateefahCamacho) [reply recipe/index.html]: Good afternoon, that's a good question the link is to Replit i will check to see if I need it

**Comment by LateefahCamacho** [comment]
Removed the additional <script> tag it was leading to Replit logo on the web page.

**Comment by nmestrada** [comment]
The updates look good to me! Feel free to merge this PR

**Pull request merged**